Cargo.toml

```diff
@@ -113,7 +113,7 @@ zeroize = { version = "1.5.7", default-features = false, features = ["zeroize_de
 
 # Oxide forks and repos
 dice-mfg-msgs = { git = "https://github.com/oxidecomputer/dice-util", default-features = false }
-gateway-messages = {git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["smoltcp"] }
+gateway-messages = {git = "https://github.com/oxidecomputer/management-gateway-service", default-features = false, features = ["smoltcp"], branch = "ignition-integration" } # FIXME
```

Comment:
Before merging this PR, switch this back to `main` once https://github.com/oxidecomputer/management-gateway-service/pull/22 is merged.